### PR TITLE
Labels given to pull request#90

### PR DESCRIPTION
**Pull request by madmaverickminion**
.labels() method added in class pull_request.py

### CODE REVIEW

File: src/scrape_up/github/pull_request.py [hunk 1]
```diff
@@ -13,6 +13,15 @@ def __scrape_page(self):
         data = requests.get(f"https://github.com/{self.username}/{self.repository}/pull/{self.pr_number}")
         data = BeautifulSoup(data.text,"html.parser")
         return data
+    def labels(self):
+        labels_found=[]
```
Comment (nikhil25803) [inline src/scrape_up/github/pull_request.py:17]: Include this method inn `documentation.md` file @madmaverickminion

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/pull_request.py (hunk 1 above)
Comment (madmaverickminion) [reply src/scrape_up/github/pull_request.py:17]: Done!

File: src/scrape_up/github/pull_request.py [hunk 2]
```diff
@@ -13,6 +13,15 @@ def __scrape_page(self):
...
+        data=self.__scrape_page()
+        label_raw=data.find_all("a",class_="IssueLabel hx_IssueLabel width-fit mb-1 mr-1")
+        for d in label_raw:
+            pass
+            labels_found.append(d.get_text().strip())   
+        return labels_found
+
```
Comment (nikhil25803) [inline src/scrape_up/github/pull_request.py:24]: Add this in  `try-excpet` block as well, to handle any unwanted error.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/pull_request.py (hunk 2 above)
Comment (madmaverickminion) [reply src/scrape_up/github/pull_request.py:24]: Okay. What should I add in except block? Can you please let me know what exceptions might occur.

Comment (nikhil25803) [reply src/scrape_up/github/pull_request.py:24]: Scrapper might not be able to fetch the data, or any other unwanted issue.

Comment (madmaverickminion) [reply src/scrape_up/github/pull_request.py:28]: Done! Kindly review it

**Review by nikhil25803 — APPROVED** [review]

**Comment by nikhil25803** [comment]
Great @madmaverickminion

**Pull request merged**